pubspec.yaml

```diff
@@ -1,23 +1,24 @@
-name: trufi_app
-description: Trufi App
+name: core
```

Comment:
Is this really supposed to be named core or rather trufi core?

Reply:
I've tested also `trufi_core` but then it generates `app.trufi.trufi_core` as package name instead of `app.trufi.core`. I want to change the name back to `trufi_core` but keep `app.trufi.core` as package name in the next step.

Reply:
From now on the name of the plugin is `trufi_core`.